### ISSUE TITLE
Provenance metadata as mitigation approach

### ISSUE THREAD

**Issue by chrisn** [w3c/ai-web-impact#4]
Section 4.1.1 says "We do not know of any credible solution". For images and video, C2PA provides a solution, which has some significant industry backing, although it requires those creating content through AI to opt in to using it (which some have done, e.g., [OpenAI](https://help.openai.com/en/articles/8912793-c2pa-in-dall-e-3)), and platforms using the information (see [Meta's announcement](https://about.fb.com/news/2024/02/labeling-ai-generated-images-on-facebook-instagram-and-threads/)).

A potential exploration for W3C could be the role user agents might play in surfacing provenance information in images and video. Currently this is done by websites, e.g., [publishers](https://www.bbc.co.uk/mediacentre/2024/content-credentials-bbc-verify) or [independent verifiers](https://contentcredentials.org/verify), but integrating this capability into the user agent could make the information more convenient for users to access, as well as being independent of any particular publisher.

**Comment by dontcallmedom** [comment]
my understanding of C2PA is that it allows to embed signed claims into media files on their sources, including potential ML-based processing; but as far as I understand, it doesn't provide a mechanism that prevents removing these claims, and in particular, wouldn't prevent someone with nefarious intent to present synthetic content as a human creation. Is that correct? this is what I was trying to convey in terms of "no credible solution".

+1 on adding the potential role of UA in surfacing provenance information

**Comment by chrisn** [comment]
Yes, claims can be removed, and C2PA is also not a "truth" mechanism, it's a provenance mechanism, so someone can make untrue claims but these are traceable to who made them, and then they take on the associated reputational risk.

I'll think about a proposed wording change ... I think "no credible solution that could guarantee" is maybe too strong as there are solutions that well-intentioned actors can use if they choose to.

I'll also create a PR for the UA aspect.
